fendermint/vm/topdown/src/lib.rs

```diff
@@ -45,6 +47,30 @@ pub struct Config {
     pub exponential_back_off: Duration,
     /// The max number of retries for exponential backoff before giving up
     pub exponential_retry_limit: usize,
+    /// The minimal number of blocks one should make the topdown proposal
```

Comment:
As discussed sync, something that I am wondering here is if this `min_proposal_interval` needs to be synced among all of the validators. What happens if one decides to set this value to `5` and the other to `100`? As long as both have the finality information for the proposal it should be fine, but it may remove the initial purpose of using this approach. WDYT?
```suggestion
    /// The minimum number of blocks one should make the topdown proposal
```

Reply:
Unless we keep it constant and we don't let users freely configure this value as I see that you are doing above.